scalajs/src/main/scala/clue/js/FetchJSBackend.scala

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package clue.js
+
+import cats.effect._
+import cats.syntax.all._
+import clue._
+import clue.model.GraphQLRequest
+import clue.model.json._
+import io.circe.syntax._
+import org.http4s.Headers
+import org.http4s.Uri
+import org.scalajs.dom.Fetch
+import org.scalajs.dom.HttpMethod
+import org.scalajs.dom.RequestInit
+import org.scalajs.dom.{ Headers => FetchHeaders }
+
+import scala.concurrent.ExecutionContext.Implicits._
+import scala.scalajs.js.URIUtils
+import scala.util.Failure
+import scala.util.Success
+
+sealed trait FetchMethod extends Product with Serializable
+object FetchMethod {
+  case object GET  extends FetchMethod
+  case object POST extends FetchMethod
+}
+
+final class FetchJSBackend[F[_]: Async](method: FetchMethod) extends TransactionalBackend[F] {
+  def request(
+    uri:     Uri,
+    request: GraphQLRequest,
+    headers: Headers
+  ): F[String] =
+    Async[F].async_ { cb =>
+      val headersʹ = new FetchHeaders()
+      headers.headers.foreach(h => headersʹ.append(h.name.toString, h.value))
```

Comment:
I suppose you can pass an auth token in the headers

Reply:
Should be the case, same as before.